src/modules/clear-completed-task.js

```diff
@@ -0,0 +1,17 @@
+import resetIndex from './index-resetter.js';
+
+export default function clearCompleted(tasks) {
+  const tempArray = [];
+  // get completed tasks indexes
+  for (let i = 0; i < tasks.length; i + 1) {
+    if (tasks[i].completed) {
+      tempArray.push(tasks[i].index);
+    }
+  }
+
+  // delete values
+  for (let i = 0; i < tempArray.length; i + 1) {
+    tasks = tasks.filter((element) => element.index !== tempArray[i]);
+  }
```

Comment:
- Hi, After cross-checking your App, I realized that the Deleting is not working. 
- Kindly solve the issue for us to check if Deleting task updates all remaining items' indexes as stated in the project requirement
![Capture](https://user-images.githubusercontent.com/63972670/206745416-30b59352-7e54-4227-8d4a-f71c0b1cfdec.PNG)
-

![Capture](https://user-images.githubusercontent.com/63972670/206746017-d100a964-4240-4ad2-8670-ae107e8132e2.PNG)